stock_backend/stock_data.py

```diff
@@ -21,9 +21,10 @@
 firebase_request_adapter = requests.Request()
```

Comment:
This file isn't updated.

Reply:
@jack-y-xu same here

Reply:
old version worked new version didn't

Reply:
I just tested it with the new version and deployed. it works